Adding support to transform TOC link

amitkumariiit opened the pull request:
Adding functionality in toc to apply a transformation to the link
 
By default link is as  #{header}. Using this function consumer can override this functionality and choose what they want as a link.

amitkumariiit committed:
adding support to transform link

amitkumariiit committed:
minor update

martinlissmyr commented:
Hi. Nice of you to contribute... 

Can you add some more documentation for the method such as signature and expected retyrnb value etc? See `format` in README.

Could you also add a test?

I would be happy to merge and release if you fix those. 👍

amitkumariiit committed:
adding test and updated readme

amitkumariiit commented:
Hi @martinlissmyr thanks for the quick check on the PR and also for your suggestions. 
I have updated the readme section and also added the test. 
Please let me know if I need to add something more.

amitkumariiit commented:
Hi @martinlissmyr can you please review the PR. We are actually waiting for this PR to be merged for some of our features in our product. I would be great if you can merge this. 
If something is needed from my side, please let me know, I would love to update.

martinlissmyr commented:
There. Merged and released via NPM. Thanks for contributing!

amitkumariiit commented:
Thanks @martinlissmyr
